src/openrct2/title/TitleSequence.h

```diff
@@ -32,6 +32,7 @@ typedef struct TitleCommand
         uint8 Zoom;         // ZOOM
         uint8 Speed;        // SPEED
         uint16 Milliseconds;      // WAIT
+        utf8 Scenario[64];        // LOADSC
```

Comment:
Is 64 arbitrary or some limit?

Reply:
64 is arbitrary, but also what is used in other areas. Currently the longest built-in title sequence name is 50 characters long exluding the null character at the end.

Reply:
We should have a `#define` for this limit then.

Reply:
Added

Reply:
It's nice you've added the `#define`, but you're still hardcoding the 64 on line 37. :wink:

Reply:
My mistake. I swear I added that in there. Ah well.